pyproject.toml

```diff
@@ -178,10 +178,10 @@ filterwarnings = [
   "error:::napari", # turn warnings from napari into errors
   "error:::test_.*", # turn warnings in our own tests into errors
   "default:::napari.+vendored.+",  # just print warnings inside vendored modules
-  "ignore:Restart required for this change:UserWarning:napari",  # triggered by a lot of async tests
   "ignore::DeprecationWarning:shibokensupport",
   "ignore::DeprecationWarning:ipykernel",
   "ignore::DeprecationWarning:tensorstore",
+  "ignore:Layer.loaded is deprecated:DeprecationWarning:",
```

Comment:
The comment above suggests that napari deprecation warnings should not go here. I added individual filters to tests related to this deprecation. But we should probably use `pytest.warns` as the comment suggests so it can be removed when not needed.

Another solution (which I probably prefer for this PR) is to not deprecate `loaded` at all - it's not a publicly settable property and we can always return `True` correctly. What do you think?

Reply:
I'll leave the decision to you here.